packages/usehooks-ts/src/useCopyToClipboard/useCopyToClipboard.demo.tsx

```diff
@@ -3,14 +3,13 @@ import { useCopyToClipboard } from './useCopyToClipboard'
 export default function Component() {
   const [copiedText, copy] = useCopyToClipboard()
 
-  const handleCopy = (text: string) => () => {
-    copy(text)
-      .then(() => {
-        console.log('Copied!', { text })
-      })
-      .catch(error => {
```

Comment:
The `copy` method never throws a rejected Promise. 
We should let the user determine if the copy was successful by the resolved value.